spec/models/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  let(:employee) { create(:employee) }
+  let(:expensive_reward) { create(:reward, price: 100) }
+
+  # rubocop:disable RSpec/ImplicitExpect
+  describe 'validations' do
+    it { should belong_to(:employee) }
+    it { should belong_to(:reward) }
+
+    it 'is expected check that employee can afford reward' do
```

Comment:
basically, it's just a suggestion: couldn't it be tested in a spec system? it would be more legible, but maybe only for me because - a suggestion

Reply:
I would also consider moving it to system spec

Reply:
I have added this test in system spec. 

But isn't it a good place to test it too since validations are part of model responsibilities? Does it being custom validation changes anything here?